tests/test_edgeql_expressions.py

```diff
@@ -1357,8 +1357,8 @@ async def test_edgeql_expr_valid_order_06(self):
                 SELECT A ORDER BY A;
             ''',
             [
-                "19:01:22.306916",
-                "20:01:22.306916",
+                "PT19H1M22.306916S",
```

Comment:
Please add `cal::relative_delta` to the `VALUES` `dict` here to cover the standard operators you've added.

Reply:
By "here" I mean this file.

Reply:
added